modules/recruiting/storage/main.tf

```diff
@@ -0,0 +1,62 @@
+resource "azurerm_resource_group" "tikjob_rg" {
+  name     = "tikjob-${var.env_name}-rg"
+  location = var.resource_group_location
+}
+
+resource "random_password" "db_password" {
+  length           = 30
+  special          = true
+  override_special = "_%@"
+}
+
+resource "azurerm_mysql_server" "tikjob_mysql" {
+  name                = "tikjob-ghost-database"
+  location            = azurerm_resource_group.tikjob_rg.location
+  resource_group_name = azurerm_resource_group.tikjob_rg.name
+
+  administrator_login          = var.ghost_db_username
+  administrator_login_password = random_password.db_password.result
+
+  sku_name   = "B_Gen5_1"
+  storage_mb = 5120 #Max size in MB
+  version    = "5.7"
+
+  auto_grow_enabled                 = true
+  backup_retention_days             = 7
+  geo_redundant_backup_enabled      = false
+  infrastructure_encryption_enabled = false
+  public_network_access_enabled     = true
+  ssl_enforcement_enabled           = true
+  ssl_minimal_tls_version_enforced  = "TLS1_2"
+}
+
+resource "azurerm_mysql_database" "tikjob_mysql_db" {
+  name                = "ghost"
```

Comment:
Seems like a bit inconsistent naming, though I've no suggestions for alternative...

Reply:
The Ghost container defaults to a database named "ghost". Tbh it could be something else but this way it is consistent with Ghost own documentation and other guides.

Reply:
I think it would make sense to rename it now. We may need to consolidate DB servers etc. in the future depending on how hard we are pressing the Azure budget, and renaming it now to e.g. `tikjob_${var.env_name}_ghost` would make that transition infinitely easier in the future.

Reply:
Name changed to suggested in https://github.com/Tietokilta/tietokilta.fi-infra/pull/12/commits/51fe870d36f230e6b80a6c19f46f3539ee195c39. The front container is always flexible for name changes and can be restarted with new name if case may be in the future, it is just a task of renaming/migrating the db.